### ISSUE TITLE
Messenger (dark mode) - Text bubbles are nearly unreadable

### ISSUE THREAD

**Issue by nexxai** [ferdium/ferdium-app#1630]
### Avoid duplicates

- [X] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-app/blob/HEAD/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issues tracker](https://github.com/ferdium/ferdium-app/issues?q=is%3Aopen+is%3Aissue+label%3A%22bug+%3Abeetle%3A%22) for a bug report similar to mine, in vain

### Ferdium Version

6.7.1

### What Operating System are you using?

macOS

### Operating System Version

macOS Sonoma 14.4

### What arch are you using?

arm64 (including Apple Silicon)

### Last Known Working Ferdium version

6.7.1

### Expected Behavior

The text in the text bubbles to be readable

### Actual Behavior

The text is light grey on white

### Steps to reproduce

1. Open Messenger
2. Attempt to read message

### Screenshots

<img width="244" alt="Screenshot 2024-03-19 at 4 41 23 PM" src="https://github.com/ferdium/ferdium-app/assets/4316564/82c080a4-2947-4c45-86e1-6fdbd56a1627">

**Comment by backd00rbenny** [comment]
Seeing the same in Arch Linux. Tried manually adjusting the css file, but had absolutely no effect. I'm wondering if Messenger have changed the attribute name for the incoming messsage background colour.

**Comment by Alphrag** [comment]
Yes this is coming from a change on Messenger's side. A hacky way to fix that at the moment is to modify the file `darkmode.css` (you can access its location when editing your messenger service) and to change line 4 to
```css
.__fb-light-mode .x1n2onr6 {
```
If you want to be more precise, you can simply change the value of the variable `--chat-incoming-message-bubble-background-color` in the css class `.x1n2onr6` which is now forcing the background colour to be white instead of respecting the value set in `.__fb-light-mode` (it purely overrides it).

If anyone else has a better way to deal with that, feel free to propose your changes, mine is very rough and just to get rid of the problem quickly :wink:

**Comment by backd00rbenny** [comment]
Nice work @Alphrag - looks to have done the trick. Figured it had to be something like that.

**Comment by YooperKirks** [comment]
I hoped for a quick fix like this @Alphrag , but sadly not working for me.

**Comment by superuser7777** [comment]
https://github.com/ducfilan/Dark-mode-Franz-Ferdi/blob/master/messenger/darkmode.css

**Comment by PlaceboPRS** [comment]
> https://github.com/ducfilan/Dark-mode-Franz-Ferdi/blob/master/messenger/darkmode.css


That is a thing of beauty! Finally I can read messages in messenger, thank you so much :)
